### DISCUSSION TITLE
2.7.0 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#3602] [讨论]
· Mod 管理优化
　　支持更新 Mod（#102）
　　支持查看可更新的 Mod 的更新日志
　　支持滑动鼠标快速选中、取消选中多个 Mod（#2333，@Chiloven945）
　　大幅加快 Mod 列表加载速度（#3037，@computer-drive）
　　Mod 作者未正确填写信息时不显示这些信息（#3470，@mc-124）
　　不再尝试从 .class 文件中获取 Mod 信息
· Mod / 整合包 下载优化
　　部分 Mod / 整合包 的文件列表中会追加一部分文件
　　下载 Mod 文件时会进行 Hash 校验
· 将 JVM 参数长度限制提高到 4000 字符（#3422，@596wjr）
· 为切换音频设备导致的错误添加提示（#3507，@youzi-2333）
· 修改关于页面的作者描述（#3583，@Silverteal）
· 支持归类 2024 愚人节版本（#3591，@ZmdNotFound，by @allMagicNB）
· 优化字符串处理、读写 ini 文件、搜索、枚举的性能
· 更新 2024 年 3 月赞助者名单，感谢大家的支持啦！
· 修复了以下 11 个 Bug（包含 1 个严重 Bug）
　　[严重] MC 24w14a+ 现要求使用 Java 21（#3601，@JingHai-Lingyun）
　　版本列表中不会显示下载过程被中断的版本（#2781，@orangeZSCB）
　　音乐播放设备变更时会持续报错（#3277，@JJJJGGGG1）
　　极个别电脑无法打开 PCL（#3467，@LBZiXL）
　　Configured Mod 错误标记为了 Forge 版（#3472，@METzX）
　　无法使用 Alibaba Dragonwell 21 Java（#3493，@penjun71）
　　无法下载部分快照版的 Mod（#3599，@xxtg666）
　　如果 Mod 联网信息中的一部分获取失败，无法在短时间内重新加载
　　Mod 信息若有本地缓存则永远不会进行联网更新
　　Mod 管理页面的加载进度可能从 100% 倒退回 99%
　　Mod 版本号如果只有一个数字，可能会显示为错误的内容

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.7.0'>2.7.0</a>.</em>

**Comment by allMagicNB** [comment]
两年啊，两年！终于做完了……

**Comment by Ad-closeNN** [comment]
啊？3:48？

**Comment by kang-png** [comment]
一直重置options怎么办
![4FEX(%JOXZN4N4_U7%S _LJ](https://github.com/Hex-Dragon/PCL2/assets/83685506/6b22ced5-3b22-4bde-970b-b8178864f9ee)

**Reply by zkitefly** [reply]
https://github.com/Hex-Dragon/PCL2/issues/3616

**Reply by MoYuan-CN** [reply]
等待下一次更新修复哦

**Reply by wuliaodexiaoluo** [reply]
有没有人 #3635  关一下
